pkg/deployment/deployment.go

```diff
@@ -0,0 +1,379 @@
+/*
+Copyright © 2025 SUSE LLC
+SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deployment
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/suse/elemental/v3/pkg/sys"
+)
+
+const (
+	EfiLabel = "EFI"
+	EfiMnt   = "/boot/efi"
```

Comment:
ESP should be mounted at `/boot` (or `/efi`) according to [BLS](https://uapi-group.org/specifications/specs/boot_loader_specification/#mount-points)